src/gen_pfc.c

```diff
@@ -289,7 +289,7 @@ static int _get_bintree_syscall_num(const struct pfc_sys_list *cur,
 	}
 
 	if (cur == NULL)
-		return -EINVAL;
+		return -EFAULT;
```

Comment:
Unrelated to this change - I noticed that `_gen_pfc_arch()`can return a negative return code, but we're ignoring the return code entirely in `gen_pfc_generate()`.  Not a big deal currently, but probably a nitpick worth fixing someday

Reply:
Oh, and `-EFAULT` isn't in `_rc_filter()`, but as mentioned in the comment above, these errors are being suppressed entirely.  So not a big deal at the moment.

Reply:
> I noticed that _gen_pfc_arch()can return a negative return code, but we're ignoring the return code entirely in gen_pfc_generate(). Not a big deal currently, but probably a nitpick worth fixing someday.

Seems related to the discussion above, I'm going to mark this comment resolved in favor of discussion it up there.

> Oh, and -EFAULT isn't in _rc_filter() ...

Well, since the default is `-EFAULT` it doesn't need to be :)

>